.github/workflows/create-cherry-pick-pr.yml

```diff
@@ -26,3 +26,4 @@ jobs:
     with:
       target_branch: ${{ inputs.target_branch }}
       pr_number: ${{ inputs.pr_number }}
+      token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is the token necessary? 🤔 

This is a workflow call, shouldn't it pick the token from the parent?

Reply:
it should, but apparently this was not working for the core repo, so olivier did add a change to the workflow that made this necessary
it is just a quick and ugly fix

Reply:
It's not working in X either. The problem is that GitHub Actions don't run on PRs created by GitHub Action of the token used is the one provided by default.

Reply:
@JCQuintas @oliviertassinari here is a better explanation of this with actual examples. It is entirely stupid, but we need to use a PAT for this! 🤦🏼

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/triggering-a-workflow#triggering-a-workflow-from-a-workflow

Reply:
Do we have a bot account we can store the PAT on?

Reply:
Yes, I plan to create that token.